WORKSPACE

```diff
@@ -0,0 +1 @@
+workspace(name = "com_googlesource_code_gflags")
```

Comment:
Bazel has a way to import another Bazel-based projects. Writing something like below in the WORKSPACE will cause that project to import gflags. Of course, we can manually copy gflags into a subdir of another project, but I don't think that's Bazel's recommended way of composing projects.

Also, RE2 has a WORKSPACE file of this form. It'd be nice to have a consistent policy within a company :)


git_repository(
    name = "com_googlesource_code_gflags",
    remote = "https://github.com/gflags/gflags.git",
    tag = "v1.0.0",
)

bind(
    name = "gflags",
    actual = "@com_googlesource_code_gflags//:gflags",
)